.eslintrc.js

```diff
@@ -27,11 +33,17 @@ module.exports = {
         zones: [
           {
             target: ['(public|server)/**/*'],
-            from: ['../../packages/**/*','packages/**/*']
+            from: ['../../packages/**/*','packages/**/*'],
           },
-        ]
-      }
-    ]
+        ],
+      },
+    ],
+    // Add cypress specific rules here
+    'cypress/no-assigning-return-values': 'error',
+    'cypress/no-unnecessary-waiting': 'error',
+    'cypress/assertion-before-screenshot': 'warn',
+    'cypress/no-force': 'warn',
+    'cypress/no-async-tests': 'error',
```

Comment:
are these standard rules? should we point all to error?

Reply:
I think this is the standard rule, for `assertion-before-screenshot` and `no-force`, I think `warn` should be fine